apps/ttd-app/src/components/ProvenanceDrawer.css

```
@@ -0,0 +1,195 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* © James Ross Ω FLYING•ROBOTS <https://github.com/flyingrobots> */
+
+.provenance-drawer {
+  position: fixed;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: 400px;
+  background: var(--bg-secondary);
+  border-left: 1px solid var(--border-color);
+  box-shadow: -4px 0 20px rgba(0, 0, 0, 0.3);
```

Stylelint (17.2.0)

[error] Expected "0.3" to be "30%" (alpha-value-notation)

(alpha-value-notation)